schema/attribute/physical-address.json

```diff
@@ -1,35 +1,37 @@
 {
-  "id": "http://platform.selfkey.org/schema/attribute/physical-address.json",
-  "$schema": "http://platform.selfkey.org/schema/identity-attribute.json",
-  "title": "Address",
-  "description": "An individual's address.",
-  "type": "object",
-  "properties": {
-    "post-office-box": {
-      "type": "string"
-    },
-    "extended-address": {
-      "type": "string"
-    },
-    "street-address": {
-      "type": "string"
-    },
-    "locality": {
-      "type": "string"
-    },
-    "region": {
-      "type": "string"
-    },
-    "postal-code": {
-      "type": "string"
-    },
-    "country-name": {
-      "type": "string"
-    }
-  },
-  "required": [ "locality", "region", "country-name" ],
-  "dependencies": {
-    "post-office-box": [ "street-address" ],
-    "extended-address": [ "street-address" ]
-  }
+	"$id": "http://platform.selfkey.org/schema/attribute/physical-address.json",
+	"$schema": "http://platform.selfkey.org/schema/identity-attribute.json",
+	"idAttribute": true,
+	"idAttributeRepository": "http://platform.selfkey.org/repository.json",
+	"title": "Address",
+	"description": "An individual's address.",
+	"type": "object",
+	"properties": {
+		"post-office-box": {
```

Comment:
Addresses are **hard**.

Let's remove `post-office-box`, `extended-address` and `street-address`, replacing them with `address-line-1`, `address-line-2` and `address-line-3`.

Reply:
great. it will simplify staff

Reply:
fixed